Dockerfile

```diff
@@ -1,52 +1,63 @@
-FROM elixir:1.13
+FROM almalinux:8.6
 
 ENV USER_HOME /home/docker
+ENV LANG en_US.UTF-8
 
-RUN addgroup --gid 1000 docker \
- && adduser --uid 1000 --gid 1000 --disabled-password --gecos "Docker User" --home ${USER_HOME} docker \
+# Ensure locale is UTF-8
+RUN dnf install --assumeyes \
+    glibc-langpack-en \
+    glibc-locale-source \ 
+ && localedef --force --inputfile=en_US --charmap=UTF-8 en_US.UTF-8 \
+ && echo "LANG=en_US.UTF-8" > /etc/locale.conf \
+ && dnf clean all \
+ && rm -rf /var/cache/dnf /var/cache/yum
+
+# Create non-root user
+RUN groupadd --gid 1000 docker \
+ && adduser --uid 1000 --gid 1000 --home ${USER_HOME} docker \
  && usermod -L docker
 
-# Configure apt, install updates and common packages, and clean up apt's cache
-ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get update \
- && apt-get upgrade --assume-yes \
- && apt-get autoremove --assume-yes \
- && apt-get install --assume-yes --no-install-recommends \
-    apt-utils \
-    apt-transport-https \
+# Install EPEL and base packages
+RUN dnf install --assumeyes glibc-langpack-en \
+ && dnf install --assumeyes https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm \
+ && dnf install --assumeyes dnf-plugins-core \
+ && dnf config-manager --set-enabled powertools \
+ && dnf update --assumeyes \
+ && dnf install --assumeyes \
     ca-certificates \
-    software-properties-common \
-    locales \
- && apt-get install --assume-yes --no-install-recommends \
     curl \
-    psmisc \
     git \
-    build-essential \
-    python \
     jq \
-    nodejs \
+    python36 \
     npm \
-    ncat \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/ \
- && update-ca-certificates
+    nmap \
+    psmisc \
+    procps-ng \
+    wget \
+ && dnf clean all \
+ && rm -rf /var/cache/dnf /var/cache/yum
 
-# Ensure locale is UTF-8
-ENV LANG       en_US.UTF-8
-ENV LC_ALL     en_US.UTF-8
-ENV LC_TYPE    en_US.UTF-8
-ENV LANGUAGE   en_US.UTF-8
-RUN echo 'en_US.UTF-8 UTF-8' >> /etc/locale.gen \
- && locale-gen \
- && dpkg-reconfigure locales
+ #Install nodejs and enable module nodejs:16
+RUN dnf module reset --assumeyes nodejs \
+ && dnf module enable --assumeyes nodejs:16 \
+ && dnf module install --assumeyes nodejs:16 
+
+#Install elixir version 1.13.0 and erlang version 25.0.3
+RUN wget https://packages.erlang-solutions.com/erlang/rpm/centos/8/x86_64/elixir_1.13.0-1~centos~8_all.rpm \
+ && wget https://packages.erlang-solutions.com/erlang/rpm/centos/7/x86_64/esl-erlang_25.0.3-1~centos~7_amd64.rpm \
```

Comment:
Is there a centos 8 build for erlang available?  Just ctrl + f ing through the index page didn't show anything for erlang 25 but there was a lot of different packages that I have no idea what is lol

Reply:
I wasn't able to find one there specifically but we could go with version 24.1.7. When I tried to install elixir by itself it cried for erlang version 24 and above, so I can grab that one from the Centos8 build list.

Reply:
@FreedomBen Done